docs/src/dataDisplay/banner.js

```diff
@@ -469,20 +468,17 @@ export default class ViewsBanner extends React.Component {
     }
 
     _renderBanners() {
-        return _.map(this.state.acData, d => {
-            return (
-                <Banner
-                    id={d.id}
-                    isOpen={d.isOpen}
-                    key={`banner-key-${d.id}`}
-                    message={d.message}
-                    onAfterClose={this._onAfterClose.bind(this)}
-                    onClose={this._onAcBannerClick.bind(this, d.id)}
-                    title={d.title}
-                    type={d.type}
-                />
-            );
-        })
+        return _.map(this.state.acData, (d) => (
```

Comment:
TOTAL NAB:
I wonder if we should consider fixing up Lodash imports in React CM UI as well.
Maybe if/when we upgrade Webpack, so that the optimizer can do its job properly.

Reply:
Yep, I think that would be a good idea that would be easy to implement.

Reply:
I kind of like the approach of doing it while we're doing other work on various files.  It's not too hard.  Obviously mistakes can be made that don't get caught during code review and escape into released code (this is why we also need to be doing unit tests! :-) so there is risk of regression.  But I think it's worth it to take some risk if, in the long term, this will help us optimize.